project-base/storefront/components/Blocks/OrderSummary/SingleProduct.tsx

```diff
@@ -14,11 +14,14 @@ export const SingleProduct: FC<SingleProductProps> = ({ item }) => {
 
     return (
         <li className="flex items-center border-b border-creamWhite py-3">
-            <Image
-                alt={item.product.mainImage?.name || item.product.fullName}
-                className="mr-4 h-14 w-14"
-                image={item.product.mainImage}
-            />
+            <div className="mr-4 flex h-14 w-14 items-center">
```

Comment:
Would it make sense to add `wrapperClass` to the Image component and then use it like this?

```
<div class={wrapperClass}>
    <Image
        alt=...
        ...
    />
</div>
```

so you do not have to wrap the Image component into a div everywhere?

Reply:
ok, makes sense :+1: